lib/codemessage.rb

```diff
@@ -40,7 +40,7 @@ def eql?(other)
     (self <=> other) == 0
   end
 
-  def <=> (other)
+  def <=>(other)
```

Comment:
Ruby 2.4+ warns about spaces after the method name and before a parenthesized argument list.